docker/py3ews/Dockerfile

```diff
@@ -5,4 +5,14 @@ COPY requirements.txt .
 RUN apk --update add --no-cache libxslt-dev \
   && apk --update add --no-cache --virtual .build-dependencies python3-dev gcc build-base wget git libffi-dev openssl-dev python3-dev libxml2-dev \
   && pip install --no-cache-dir -r requirements.txt \
-  && apk del .build-dependencies
\ No newline at end of file
+  && apk del .build-dependencies
+
+# Handling the issue described here: https://github.com/ecederstrand/exchangelib/issues/608
+# by activating the legacy sect in openssl.cnf (re-enableing md4 to hashlib)
+RUN cp /etc/ssl/openssl.cnf /etc/ssl/openssl.cnf.org \
+&& sed -i '/default = default_sect/a legacy = legacy_sect' /etc/ssl/openssl.cnf \
+&& echo -e 'activate = 1\n\n[legacy_sect]\nactivate = 1\n' > /tmp/ssl.cnf \
+&& sed -i '/ activate = 1/r /tmp/ssl.cnf' /etc/ssl/openssl.cnf \
```

Comment:
where is this original
`activate = 1` ?

Reply:
I deleted it in line 16 (if that is what you are asking).

Reply:
in line 15 you replace activate=1 with the contents of the  /tmp/ssl.cnf file, which is activate = 1\n\n[legacy_sect]\nactivate = 1\n

where is the original activate = 1 that youre appending [legacy_sect]\nactivate = 1 to?